src/action.ts

```diff
@@ -28,6 +32,17 @@ export async function run(): Promise<void> {
 		});
 		// Set the built releaseId in the output
 		core.setOutput('release_id', releaseId);
+
+		if (releaseId) {
+			rawVersion = await balena.getReleaseVersion(parseInt(releaseId, 10));
+		}
+
+		core.setOutput('raw_version', rawVersion);
+
+		if (rawVersion && core.getBooleanInput('create_ref', { required: false })) {
```

Comment:
if we make getReleaseVersion throw when it fails to get the rawVersion then we don't have to check if it's set (I'm going to write a comment about this function that it should throw)

```suggestion
		if (core.getBooleanInput('create_ref', { required: false })) {
```